paasJobs/jobs/s3_upload.sh

```diff
@@ -59,7 +61,7 @@ local job_dir="$LOCAL_TMP_DIR"
     s3 \
     --s3-raw-ingest-prefix="$s3_raw_ingest_prefix" \
     --s3-parsed-ingest-prefix="$s3_parsed_ingest_prefix"
```

Comment:
add a slash at the end of this like and you can take it off at the end of the metadata-creation-group line